package.json

```diff
@@ -273,5 +273,6 @@
     "node-fetch": {
       "whatwg-url": "^14.0.0"
     }
-  }
+  },
+  "packageManager": "pnpm@10.13.1+sha256.0f9ed48d808996ae007835fb5c4641cf9a300def2eddc9e957d9bbe4768c5f28"
```

Comment:
What's this?

Reply:
An oopsie from accidentally using `pnpm` locally. It's sooooo much better than `npm` but I didn't mean to introduce it here. Reverted.